src/ducks/settings/AccountsSettings.jsx

```diff
@@ -130,7 +130,7 @@ class AccountsSettings extends Component {
 }
 
 const mapStateToProps = state => ({
-  collectUrl: getAppUrlById(state, 'io.cozy.apps/collect')
+  collectUrl: getHomeURL(state)
```

Comment:
Does it require lot of changes to rename this `collectUrl` to `homeUrl`?

Reply:
This component did not use collectUrl anymore :tada: